.circleci/config.yml

```diff
@@ -106,10 +106,10 @@ commands:
           git push -q origin HEAD
 
 jobs:
-  checks_and_unit_tests-ubuntu2004-go116: &checksAndUnitTests
+  checks_and_unit_tests-ubuntu2204-go118:
     machine:
       docker_layer_caching: true
-      image: ubuntu-2004:202104-01
+      image: ubuntu-2204:current
```

Comment:
This seems like a large change?  Should we pin ubuntu to a version?

Reply:
This is the way 1.18 tests are running already -- it'd be pretty stunning to see ubuntu break the most basic of functionality we use in a security update release, and I'd rather get the security updates. I understand why some folks may not wanna do that though.

Reply:
Yea for this it is probably fine.